exporter/awsprometheusremotewriteexporter/Makefile

```diff
@@ -0,0 +1 @@
+include ../../Makefile.Common
```

Comment:
Do we need Makefiles for packages?

Reply:
This is commonly found among contrib components. It allows us to run some `make` commands for individual components as some command can be very slow when running for all